filnet/router.go

```diff
@@ -0,0 +1,25 @@
+package filnet
+
+import (
+	"context"
+
+	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	pstore "gx/ipfs/QmRhFARzTHcFh8wUxwN5KvyTGq73FLC65EfFAhz8Ng7aGb/go-libp2p-peerstore"
+	routing "gx/ipfs/QmWaDSNoSdSXU9b6udyaq9T8y6LkzMwqWxECznFqvtcTsk/go-libp2p-routing"
+)
+
+// Router is used to find information about who has what content.
```

Comment:
could you make this more specific? what kind of a thing is 'who' and what kind of content are we talking about?